src/models/user.js

```diff
@@ -70,6 +82,7 @@ class User {
   }
   set firstName(newFirstName) {
     if (newFirstName) {
+      this._isDirty = true;
```

Comment:
I think that it's a bit tricky because you will need to track previous values and current values to determine the state. Since this is a minor feature that is not currently being used (it will be probable be used in the coming months), we can open this into a `refactor` issue. Is that ok with you?